### PR TITLE
Updated to Godot 4.1.1

### PR DESCRIPTION
**Pull request by adambiser**
The way to copy animations appears to be different now.

I have followed the 3D tutorial using Godot 4.1.1 and it appears to be up-to-date except for the change in this pull request.

I did notice that "v3.2.4.beta2.official" appears in the Project Manager image under "Setting up the game area".

### CODE REVIEW

**Comment by AThousandShips** [comment]
You should target `master` not `4.1`, changes can then be cherry picked, see [here](https://docs.godotengine.org/en/latest/contributing/documentation/contributing_to_the_documentation.html#contributing-changes).

If you need more information about the update process please see: [PR Workflow - Updating your branch](https://docs.godotengine.org/en/stable/contributing/workflow/pr_workflow.html#updating-your-branch).

**Comment by AThousandShips** [comment]
(accidentally closed due to mobile interface)

File: getting_started/first_3d_game/09.adding_animations.rst [hunk 1]
```diff
@@ -260,8 +260,10 @@ For example, both the ``Mob`` and the ``Player`` scenes have a ``Pivot`` and a
 ``Character`` node, so we can reuse animations between them.
 
 Open the *Player* scene, select the AnimationPlayer node and open the "float"
-animation. Next, click on **Animation > Copy**. Then open ``mob.tscn``,
-create an AnimationPlayer child node and select it. Click **Animation > Paste**
+animation. Next, click on **Edit > Copy Tracks** and check "Character.position"
+and "Character.rotation".  Click the "Copy" button.  Then open ``mob.tscn``,
+create an AnimationPlayer child node and select it.  As we did for ``Player``, click on
+*Animation -> New* and name the animation "float".  Now click **Edit > Paste Tracks**
```
Comment (AThousandShips) [inline getting_started/first_3d_game/09.adding_animations.rst:266]:
```suggestion
animation. Next, click on **Edit > Copy Tracks** and check "Character.position"
and "Character.rotation". Click the "Copy" button. Then open ``mob.tscn``,
create an AnimationPlayer child node and select it. As we did for ``Player``, click on
*Animation -> New* and name the animation "float". Now click **Edit > Paste Tracks** 
```

Comment (adambiser) [reply getting_started/first_3d_game/09.adding_animations.rst:266]: Somehow I thought I saw I saw double spaces between sentences was the style.

**Comment by AThousandShips** [comment]
You will still need to fix your branch, see the above instructions

**Comment by adambiser** [comment]
Yeah, I'm probably going to redo this because it was made directly on the github site using some of its automation.  No clue why it latched on to the 4.1 branch to begin with.

**Comment by AThousandShips** [comment]
You probably edited it from the `stable` version of the docs

**Comment by adambiser** [comment]
I'll get it squared away when I have a bit more time.  Maybe this evening.

**Pull request closed without merging**